sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1683,35 +1683,6 @@ class MapboxMap :
     isMapValid = false
   }
 
-  /**
-   * Set map projection for Mapbox map.
-   *
-   * @param mapProjection either [MapProjection.Globe] or [MapProjection.Mercator] projection that will be applied to Mapbox map.
-   */
-  @MapboxExperimental
-  override fun setMapProjection(mapProjection: MapProjection) {
```

Comment:
wonder if we should Deprecate first and then remove, it would likely break user's code? but as it is experimental, I guess it's fine to just remove.